lls/cache.c

```diff
@@ -320,10 +320,17 @@ lls_process_reqs(struct lls_config *lls_conf)
 				: lls_conf->tx_queue_back;
 			int i;
 			for (i = 0; i < arp->num_pkts; i++) {
+				struct ether_hdr *eth_hdr =
+					rte_pktmbuf_mtod(arp->pkts[i],
+						struct ether_hdr *);
+				struct arp_hdr *arp_hdr =
+					rte_pktmbuf_mtod_offset(arp->pkts[i],
+						struct arp_hdr *,
+						pkt_in_l2_hdr_len(
+							arp->pkts[i]));
 				if (process_arp(lls_conf, arp->iface,
 						tx_queue, arp->pkts[i],
```

Comment:
Defining a local variable `pkt` would avoid repeating `arp->pkts[i]` a couple of times.